File: Scripts/Listener/IInitListener.cs

// *******************************************
// Company Name:	深圳市晴天互娱科技有限公司
//
// File Name:		IInitListener.cs
//
// Author Name:		Bridge
//
// Create Time:		2024/09/03 10:32:47
// *******************************************

namespace Bridge.Common
{
	/// <summary>
	/// 
	/// </summary>
	public interface IInitListener
	{
		void OnSuccess();

		void OnError(int errCode, string errMsg);
	}
}

File: Plugins/Android/ThirdSDK.androidlib/src/main/java/com/bridge/common/listener/IInitListener.java
package com.bridge.common.listener;

/**
 * 初始化监听
 */
public interface IInitListener {
    /**
     * 初始化成功
     */
    void onSuccess();

    /**
     * 初始化失败
     * @param errCode 错误码
     * @param errMsg 错误信息
     */
    void onError(int errCode, String errMsg);
}
